ci-operator/step-registry/gather/gather-chain.yaml

```diff
@@ -2,6 +2,7 @@ chain:
   as: gather
   steps:
   - ref: gather-must-gather
+  - ref: gather-must-gather-network
```

Comment:
@pecameron can you create another gather chain in the same directory that is `gather-with-network-chain.yaml`? We don't want to gather network logs from all CI runs, just ones we use in the CNO and ovnkube (for now).

Reply:
OK